Database/Corrections/QuestieCorrections.lua

```diff
@@ -425,7 +424,8 @@ function QuestieCorrections:PreCompile(callback) -- this happens only if we are
         end
     end
 
-    timer = C_Timer.NewTicker(0.1, function()
+    while true do
+        coroutine.yield()
         for _=0,Questie.db.global.debugEnabled and 4000 or 72 do -- 72 operations per NewTicker
```

Comment:
Can you move the magic numbers 4000 and 72 into a variable?

And also remove the commented code in the `else` case?